nilearn/plotting/img_plotting.py

```diff
@@ -805,8 +809,28 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
             display.add_contours(map_img, levels=[thr],
                                  linewidths=linewidths,
                                  colors=[color], filled=filled,
-                                 alpha=alpha, linestyles='solid')
-
+                                 alpha=alpha, linestyles='solid', )
+    if colorbar:
+        display._colorbar = True
+        display._show_colorbar(matplotlib.colors.LinearSegmentedColormap.
+                               from_list('segmented colors',
+                                         color_list, n_maps + 1),
+                               matplotlib.colors.Normalize(1, n_maps + 1))
+        tick_locator = matplotlib.ticker.MaxNLocator(nbins=10)
+        display.locator = tick_locator
+        display._cbar.update_ticks()
```

Comment:
The code below would be more readable if you created an intermediate variable containing 
```
np.round(np.linspace(1, n_maps, min(n_maps, 10)))
```